Components/HeroSection.tsx

```diff
@@ -74,7 +74,7 @@ const HeroSection = () => {
               />
               <div
                 onClick={() => router.push("/courses")}
-                className="flex flex-row items-center gap-4 p-4 text-white bg-black border-2 border-black rounded-full h-fit cursor-pointer dark:bg-zinc-50"
+                className="flex flex-row items-center gap-4 p-4 text-white bg-black border-2 border-black rounded-full h-fit cursor-pointer dark:bg-zinc-50 pr-7"
```

Comment:
don't add padding to make it align in the center.

Just use flex properties as its already using `flex`